src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx

```diff
@@ -126,16 +167,16 @@ You can locate elements by their `class`, `id`, `link text`, `name` or even XPat
   >
     For more complex page structures, you can use XPath to locate the target element:
 
-    ```
-    $browser.findElement($driver.By.xpath("//input[@placeholder = 'search-query-field']"));
+    ```js
+    await $webDriver.findElement($selenium.By.xpath("//input[@placeholder = 'search-query-field']"));
     ```
 
-    XPath selectors can break when your website's structure changes. For tips on building good XPath selectors, see [Optimizing XPath selectors](https://blog.newrelic.com/engineering/optimize-xpath-selectors/). For an example of locating an element by its xpath, see [Search a website](/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring/).
+    XPath selectors can break when your website's structure changes. For tips on building good XPath selectors, see [Optimizing XPath selectors](https://blog.newrelic.com/engineering/optimize-xpath-selectors/). For an example of locating an element by its XPath, see [Locate by XPath](#locate-by-xpath).
```

Comment:
The last sentence here is another self-referential link: this _is_ the locate-by-xpath section! Should probably be removed like the others.

Reply:
Good catch!